### PR TITLE
Py3 kanghyojun rev

### PR DESCRIPTION
**Pull request by jrepp**
Submitting a PR to run travis CI

### CODE REVIEW

**Comment by kanghyojun** [comment]
Can i add `s2_cli.py` to `console_script` so that we can use `s2_cli.py` command when install `s2protocol`.


```python
    entry_points={
        'console_scripts': [
            's2protocol = s2protocol.s2protocol:main',
            's2_cli.py = s2protocol.s2_cli:main',
        ]
    },
```

And Can we remove `s2protocol` in `entry_points`?`s2protocol.main()` function is inexist.


My suggest is here https://github.com/kanghyojun/s2protocol/commit/29ec047aa1798d2477f4672c4d8ed768fca7f831

**Comment by kanghyojun** [comment]
Sadly `json.dumps()` has no keyword arugment encoding. I try to test `s2_cli` as well

**Comment by jrepp** [comment]
Thanks for the edits to s2_cli - this is the primary interface that people use to interface with other languages. Replay -> s2_cli -> json -> C# etc

I haven't spent time to familiarize myself with tox but it seems very sensible and I'm on board with what you're doing there. 

I think putting a few more calls in the tox script to get more coverage would be helpful. 

It was 2AM for me local time so I ran out of steam - hoping to get this PR merged soon. Thanks for your support.

**Comment by kanghyojun** [comment]
@jrepp Hey, I've just added 3 commits below to fix CI.

- https://github.com/kanghyojun/s2protocol/commit/96f7a5397bd023a4ce93a70e8791be31c57a5078
- https://github.com/kanghyojun/s2protocol/commit/908aff858761dbdb2ef6f164ef85d6db8298f8b8
- https://github.com/kanghyojun/s2protocol/commit/72b0fd9d45f627f8119cc42383cee6586d7e59fa

Can you merge these commit your branch & review it? [StanczakDominik](/StanczakDominik) already approved https://github.com/Blizzard/s2protocol/pull/86. But I think CLI test  is necessary.

**Comment by kanghyojun** [comment]
@jrepp I've just merged your changes & upload to https://github.com/Blizzard/s2protocol/pull/86. And CI is fixed! :)

File: s2protocol/decoders.py [hunk 1]
```diff
@@ -20,6 +20,8 @@
 
 import struct
 
+from .compat import byte_to_int
```
Comment (wroersma) [inline s2protocol/decoders.py]: This really should use 

    from s2protocol.compat import byte_to_int

**Pull request closed without merging**